backend/fms_core/viewsets/_constants.py

```diff
@@ -133,8 +133,8 @@
 _instrument_filterset_fields: FiltersetFields = {
     "id": PK_FILTERS,
     "name": CATEGORICAL_FILTERS_LOOSE,
-    "serial_id": CATEGORICAL_FILTERS_LOOSE,
-    "type__id": CATEGORICAL_FILTERS_LOOSE,
+    "serial_id": FK_FILTERS,
```

Comment:
serial_id is a string with a machine name similar to a barcode... you can keep CATEGORICAL_FILTERS_LOOSE.